src/server/ua_services_subscription.c

```diff
@@ -388,6 +388,7 @@ static void
 Operation_SetMonitoringMode(UA_Server *server, UA_Session *session,
                             struct setMonitoringContext *smc,
                             UA_UInt32 *monitoredItemId, UA_StatusCode *result) {
+    // TODO check monitoredItemType
```

Comment:
What do you mean by this comment?

Reply:
I do not know if we are concerned here, but do something like this:

https://github.com/open62541/open62541/blob/c3885fb0cebd5502df38cdf0d29fd56e098562d6/src/server/ua_subscription_datachange.c#L35-L52

to see with @aDogCalledSpot https://github.com/open62541/open62541/pull/1560

Reply:
Ah, sorry, now I got what you mean. For now I would check for the itemType and if it isn't `CHANGENOTIFY` then set the status code to BadUnsupportedService or BadNotImplemented or something similar